util/src/test/java/org/killbill/billing/DBTestingHelper.java

```diff
@@ -79,14 +79,16 @@ protected synchronized void executePostStartupScripts() throws IOException {
                                "CREATE TABLE accounts (\n" +
```

Comment:
We should brainstorm to see if we can't get rid of that: Having this hardcoded tables in our tests that end up diverging from DLL is quite painful -- and error prone -- to maintain.

Reply:
IIRC, we need it today because `util` relies on DDL provided by "downstream" modules (like account). The dependency is backward.

To get rid of it, we would probably need to rewrite some of the tests. I could look into it.